capstone/src/main/webapp/about.html

```diff
@@ -62,28 +64,32 @@ <h1>The Creators Of Perspective Hub</h1>
               </div>
               <div class="col-md-4">
                 <div class="card mb-4 shadow-sm">
-                  <img class="card-img-top" src="#" alt="Chris Mohri" style="width:100px;height:225px;"></img>
+                  <br><img class="rounded mx-auto d-block" src="assets/chris.jpg" alt="Chris Mohri" style="max-width:100px;max-height:225px;"></img><br>
```

Comment:
The `<br>` here does not appear in the other image lines - was this intentional?

Reply:
Yes, i used <br> to ensure the image was centered. It was only done for Chris's image because of the aspect ratio of his image. The other images were centered without it.

Reply:
Ok, sounds good!